vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -65,80 +63,87 @@ def sankey(data_frame: pd.DataFrame, source: str, target: str, value: str, label
     (y-axis 2), sharing the same x-axis. The y-axes for the bar and line graphs are synchronized and overlaid.
 
     Args:
         data_frame (pd.DataFrame): The data source for the chart.
-        x (str): The column name to be used for the x-axis.
-        y_column (str): The column name to be used for the y-axis 1, representing the column chart.
-        y_line (str): The column name to be used for the y-axis 2, representing the line chart.
+        x (str): Either a name of a column in data_frame, or a pandas Series or array_like object.
+        y_column (str): Either a name of a column in data_frame, or a pandas Series or array_like object.
+        y_line (str): Either a name of a column in data_frame, or a pandas Series or array_like object.
 
     Returns:
-        go.Figure: : A Plotly Figure object representing the combined column and line chart.
+        go.Figure: : A Plotly Figure object of the combined column and line chart.
 
     """
-    fig = make_subplots(specs=[[{"secondary_y": True}]])
+    # We use px.bar and px.line so that we get the plotly express hoverdata, axes titles etc. Bar is used arbitrarily
+    # selected as the "base" plot and then line added on top of it. This means manually incrementing
+    # color_discrete_sequence for the line plot so that the colors are not the same for bar and line.
+    bar = px.bar(data_frame, x=x, y=y_column)
+    fig = make_subplots(figure=bar, specs=[[{"secondary_y": True}]])
 
-    fig.add_trace(
-        go.Bar(x=data_frame[x], y=data_frame[y_column], name=y_column),
-        secondary_y=False,
-    )
-
-    fig.add_trace(
-        go.Scatter(x=data_frame[x], y=data_frame[y_line], name=y_line),
-        secondary_y=True,
-    )
-
-    fig.update_layout(
-        xaxis={"type": "category", "title": x},
-        yaxis={"tickmode": "sync", "title": y_column},
-        yaxis2={"tickmode": "sync", "overlaying": "y", "title": y_line},
+    line = px.line(
+        data_frame,
+        x=x,
+        y=y_line,
+        markers=True,
+        color_discrete_sequence=fig.layout.template.layout.colorway[len(bar.data) :],
     )
+    for trace in line.data:
+        fig.add_trace(trace, secondary_y=True)
+    fig.update_layout(yaxis2={"tickmode": "sync", "overlaying": "y", "title": line.layout.yaxis.title})
 
     return fig
 
 
 @capture("graph")
-def categorical_column(data_frame: pd.DataFrame, x: str, y: str):
-    """Creates a column chart where the x-axis values are converted to category type."""
-    fig = px.bar(
-        data_frame,
-        x=x,
-        y=y,
-    )
+def categorical_column(data_frame: pd.DataFrame, **kwargs) -> go.Figure:
+    """Creates a column chart where the x-axis values are converted to category type.
+
+    Args:
+        data_frame (pd.DataFrame): The data frame for the chart. Can be long form or wide form.
+            See https://plotly.com/python/wide-form/.
```

Comment:
```suggestion
        data_frame: The DataFrame for the chart. Can be long form or wide form.
            See https://plotly.com/python/wide-form/.
```
Was it possible to add wide and long form on the butterfly chart? If yes we should make the docs sync

Reply:
I think yes, but let me check!

Reply:
Yes indeed it is.